keymaps/ink.cson

```diff
@@ -14,8 +14,15 @@
 'ink-console':
   'shift-space': 'unset!'
 
+'.platform-win32 .ink, .platform-linux .ink':
+  'ctrl-c': 'native!'
+
+'.platform-darwin .ink':
+  'cmd-c': 'native!'
+
 'atom-text-editor:not([mini])':
   'escape': 'inline:clear-current'
+  'alt-t': 'inline-results:toggle'
```

Comment:
`C-i C-t` maybe? I kinda like an unprefixed keybinding for this, but don't have any super strong feelings about it.